src/index.html

```diff
@@ -32,20 +32,48 @@ <h1 class="heading">CROP AI</h1>
           <a href="#" class="button-Index" id="about_btn">ABOUT</a>
           <a href="#" class="button-Index" onclick="show_input_container()">ANALYZE</a>
           <div id="about_content">This is a crop advisory app for farmers and gardeners. It can diagnose pest damage, plant disease and nutrient deficiencies that are affecting crops and can offer corresponding treatment measures.</div>
-         
+
         </div>
       </div>
 
       <div id="input_container" class="hidden"></div>
-      
-      <div id="result_container" class="hidden"></div>
-    
+
+      <div id="result_container">
+        <div id="box">
+          <div id="float-child">
+            <img id= "pepper-bell" src="./img/leaf.jfif" alt="Image not found"  height= 300px width="200px">
```

Comment:
change id to crop_img (every time, it will not be bell pepper)

Add hidden class to landing_container (because only one div will be shown at one time, so develop it with that mindset)

Reply:
done